src/main.cpp

```diff
@@ -228,8 +228,8 @@ static void print_usage()
             "\t                           Where to store memory dumps\n"
             "\t --dll-hooks-list <file>\n"
             "\t                           List of DLL functions to be hooked\n"
-            "\t                           each entry in a separate line of file\n"
-            "\t                           line format: dll_name,function_name,num_of_arguments\n"
+            "\t                           example file for Win7 x64:\n"
+            "\t                           src/plugins/apimon/example/dll-hooks-list-win7x64\n"
```

Comment:
I think the extended message could be removed from printing here, what would be nice is a Wiki entry for the memdump/apimon plugin that also documents the format.

Reply:
i've removed the extended message, we will soon provide some input for a wiki (btw. it's not possible to send a regular pull request to the wiki AFAIK)

Reply:
Yea, Github's Wiki is a bit gimped, but you can git clone it from https://github.com/tklengyel/drakvuf.wiki.git, make your changes, push it to your own username & create an Issue for it pointing out your repo/branch. I can pull your branch & push to the main wiki from there. It's not as nice as just hitting the merge button but oh well.